lib/Extension/PhpCodeSniffer/Model/PhpCodeSnifferProcess.php

```diff
@@ -146,6 +146,10 @@ private function runDiagnosticts(string $url, string $text, array $options = [])
         });
     }
 
+    /**
+     * @return string Filename MUST include PHP extension, otherwise
+     * phpcs will not process it.
```

Comment:
Move the "comment" to the line above so it's not part of `@return` (alternatively could be `@param` as intended? but I'd just make it a general comment on the mehtod)